docs/release-notes/2.16.0.md

```diff
@@ -20,6 +20,13 @@
 
 Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
 
+## Important note for users of PMM 2.16.0
+
+If you started using PMM from version 2.16 and have already upgraded to 2.17, 2.18, or 2.19, then you need to perform additional steps after upgrading to 2.20.
+
+1. Enter the container: `docker exec -it pmm-server bash`
+2. Roll back `pmm2-client` package to stable version: `yum downgrade -y pmm2-client`
```

Comment:
3. `exit` to exit container.
according to Roma our user could be lost here :)

Reply:
There has to be a limit. Anyone stumped by point 1 won't be helped by adding point 3. But it's easy to do, if you want.